difference.js

```diff
@@ -0,0 +1,7 @@
+function difference(array, ...values) {
+    for (let i = 0; i < values.length; i++) {
```

Comment:
plz read https://medium.com/weekly-webtips/imperative-vs-declarative-programming-in-javascript-25511b90cdb7#:~:text=Imperative%20code%20uses%20statements%20and,computer%20how%20to%20do%20things.

you mix approaches, but function works even with high complexity